### PR TITLE
[MOBILE-1508] Release 11.0

### PR DESCRIPTION
**Pull request by BrianBatchelder**
### What do these changes do?
Major release adding support for named user and date attributes. Also updated iOS and Android SDKs.

### Why are these changes necessary?
Rolling out named user and date attribute support to all frameworks.

### How did you verify these changes?
Received pushes on Android and iOS sample apps.

### Anything else a reviewer should know?
I added `npm install` to `update_version.sh` so `package-lock.json` gets updated.

### CODE REVIEW

File: scripts/update_version.sh [hunk 1]
```diff
@@ -14,3 +14,6 @@ sed -i '' "s/\version\": \".*\",/\version\": \"$VERSION\",/g" package.json
 sed -E -i '' "s/\version=\"[0-9]+\.[0-9]+\.[0-9]+.*\"/\version=\"$VERSION\"/g" plugin.xml
 sed -E -i '' "s/\android:value=\"[0-9]+\.[0-9]+\.[0-9]+.*\"/\android:value=\"$VERSION\"/g" plugin.xml
 sed -E -i '' "s/\<string>[0-9]+\.[0-9]+\.[0-9]+.*<\/string>/\<string>$VERSION<\/string>/g" plugin.xml
+
+# Update packages
+npm install
```
Comment (rlepinski) [inline scripts/update_version.sh:19]: npm instatll does not update packages. i dont think this belongs here.

Comment (BrianBatchelder) [reply scripts/update_version.sh:19]: Huh. It updated `package-lock.json`.

Comment (rlepinski) [reply scripts/update_version.sh:19]: that version does not matter, lets leave this be.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,14 @@
 # Cordova Plugin Changelog
 
+## Version 11.0.0 - June 3, 2020
```
Comment (marc-scig) [inline CHANGELOG.md:3]: That's tomorrow, fyi

File: scripts/update_version.sh (hunk 1 above)
Comment (marc-scig) [reply scripts/update_version.sh:19]: agreed

File: CHANGELOG.md (hunk 2 above)
Comment (BrianBatchelder) [reply CHANGELOG.md:3]: I figured we'd do it in the morning.

File: scripts/update_version.sh (hunk 1 above)
Comment (BrianBatchelder) [reply scripts/update_version.sh:19]: Oh, OK. There were commits in the repo _just_ to update `package-lock.json`. I figured it must be important.

**Pull request merged**